sorc/build_ufs.sh

```diff
@@ -1,44 +1,38 @@
 #! /usr/bin/env bash
 set -eux
 
+cwd=$(pwd)
+
 # Default settings
 APP="S2SWA"
 CCPP_SUITES="FV3_GFS_v16,FV3_GFS_v16_ugwpv1,FV3_GFS_v17_p8,FV3_GFS_v16_coupled_nsstNoahmpUGWPv1,FV3_GFS_v17_coupled_p8"
 
-while getopts "a:v" option; do
+while getopts ":dav" option; do
```

Comment:
The app argument will no longer work with this construction. Option string should be `:da:v`

Reply:
fixed.